docs-chef-io/content/server/upgrades.md

```diff
@@ -46,6 +46,16 @@ Chef Infra Server 14.0 moved from Solr to Elasticsearch as its search index.
 
 The Chef Infra Server 14 upgrade does not automatically reindex existing external Elasticsearch installations.
 
+#### Upgrading to 14.13
+
+Chef Infra Server 14.13 supports External Opensearch for indexing. Migrating from Elasticsearch to External Opensearch needs ontime reindexing and reconfigure after upgrade. This process may take longer depending on your server hardware and the size of the node objects on your Chef Infra Server.
+
```

Comment:
https://docs.chef.io/server/upgrades/#standalone-server useful to give some estimates. Even better if we could provide what the estimates are based off of.